fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationSettingsFragment.kt

```diff
@@ -59,6 +74,55 @@ class TranslationSettingsFragment : Fragment(), UserInteractionHandler {
         }
     }
 
+    @Composable
+    private fun getTranslationSwitchItemList(): MutableList<TranslationSwitchItem> {
```

Comment:
nit: Could you add a comment above this that it is for the additional translations global settings between page settings and global settings? (A little difficult to name since it is between pages.)

Reply:
Done